examples/custom-resources/service-insight/README.md

```diff
@@ -0,0 +1,189 @@
+# Support for Service Insight
+
+  > The Service Insight feature is available only for F5 NGINX Plus.
+
+To use the [Service Insight](https://docs.nginx.com/nginx-ingress-controller/logging-and-monitoring/service-insight/) feature provided by NGINX Ingress Controller you must enable it by either setting `serviceInsight.create=true` in your `helm install/upgrade...` command OR  [manifest](../../../deployments/deployment/nginx-plus-ingress.yaml) depending on your preferred installation method.
+
+In the following example we'll enable the Service Insight in the NGINX Ingress Controller using [manifests (Deployment)](../../../deployments/deployment/nginx-plus-ingress.yaml):
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-ingress
+  namespace: nginx-ingress
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx-ingress
+  template:
+    metadata:
+      labels:
+        app: nginx-ingress
+        app.kubernetes.io/name: nginx-ingress
+    spec:
+      serviceAccountName: nginx-ingress
+      automountServiceAccountToken: true
+      securityContext:
+      ...
+      containers:
+      - image: nginx-plus-ingress:3.0.2
+        imagePullPolicy: IfNotPresent
+        name: nginx-plus-ingress
+        ports:
+        - name: http
+          containerPort: 80
+        - name: https
+          containerPort: 443
+        - name: readiness-port
+          containerPort: 8081
+        - name: prometheus
+          containerPort: 9113
+        - name: service-insight
+          containerPort: 9114
+        readinessProbe:
+          httpGet:
+            path: /nginx-ready
+            port: readiness-port
+          periodSeconds: 1
+        resources:
+        ...
+        securityContext:
+        ...
+        env:
+        ...
+        args:
+          - -nginx-plus
+          - -nginx-configmaps=$(POD_NAMESPACE)/nginx-config
+        ...
+          - -enable-service-insight
+
+```
+
+## Deployment
+
+[Install NGINX Ingress Controller](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/). Remember to uncomment the `-enable-service-insight` option.
+
+Enable access to the Ingress Controller. Examples below use the `nodeport` service.
```

Comment:
```suggestion
[Install NGINX Ingress Controller](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/), and uncomment the `-enable-service-insight` option: this will allow Service Insight to interact with it.

The examples below use the `nodeport` service.
```

Small change - a reader can't "remember" to do something if they were unaware they had to do it in the first place, so this makes it an imperative instruction.

Reply:
```suggestion
[Install NGINX Ingress Controller](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/). Remember to uncomment the `-enable-service-insight` option.

Enable access to the Ingress Controller. Examples below use the `nodeport` serviceType.
```